Airports/EGNO/Ownership.txt

```diff
@@ -1,11 +1,21 @@
 SECTOR:NOTWR:0:2000
 OWNER:NOT:NOR:VVC:VVW:VV
+ALTOWNER:Observing LON_N_CTR:NOT:NOA:VVC:VVW:VV:WAL:PCP:PCW:PC:O:LMEN:LN:LWN:LNC:L
```

Comment:
NOA should be NOR

Reply:
Guess I never updated my clipboard. Fixed.